tests/unit/test_errorhandler.py

```diff
@@ -10,101 +10,74 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
-from awscli.testutils import unittest
+import io
+from collections import namedtuple
+
+from botocore.exceptions import (
+    NoRegionError, NoCredentialsError, ClientError,
+    ParamValidationError as BotocoreParamValidationError,
+)
+from awscli.arguments import UnknownArgumentError
+from awscli.argparser import ArgParseException
+from awscli.argprocess import ParamError, ParamSyntaxError
+from awscli.customizations.exceptions import (
+    ParamValidationError, ConfigurationError
+)
 
-import mock
+from awscli.testutils import unittest
 from awscli import errorhandler
 
 
-class TestErrorHandler(unittest.TestCase):
+class TestChainedExceptionHandler(unittest.TestCase):
 
-    def create_http_response(self, **kwargs):
-        response = mock.Mock()
-        for key, value in kwargs.items():
-            setattr(response, key, value)
-        return response
+    @classmethod
+    def setUpClass(cls) -> None:
```

Comment:
I'm guessing the type check is coming from the IDE here. I'd say remove the type check if we are not being consistent about it.